CaseWorkDesktopTool.Frontend/Models/SigChange.cs

```diff
@@ -0,0 +1,34 @@
+﻿
```

Comment:
Please move these Models/ DTOs to the Application layer in Common/Models folder or under their own specific Entity name. Like how it is done in the template.